agrirouter-api-java-api/src/test/java/com/dke/data/agrirouter/api/factories/impl/OnboardErrorTest.java

```diff
@@ -0,0 +1,18 @@
+package com.dke.data.agrirouter.api.factories.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.dke.data.agrirouter.api.dto.onboard.OnboardingError;
+import com.google.gson.Gson;
+import org.junit.jupiter.api.Test;
+
+public class OnboardErrorTest {
+  @Test
+  public void readError() {
```

Comment:
Pattern for test methods should be `given ... do something ... should` for example `givenResultAsJsonParseErrorShouldReturnObjectContainingValidErrorCode`

Reply:
Removed, as it only "Tested itself". Maybe add a different test in the future; would require a setLastError-Function though